src/dsymbol/conversion/first.d

```diff
@@ -206,6 +206,22 @@ final class FirstPass : ASTVisitor
 		auto lookup = Mallocator.instance.make!TypeLookup(TypeLookupKind.inherit);
 		writeIotcTo(bc.type2.typeIdentifierPart, lookup.breadcrumbs);
 		currentSymbol.typeLookups.insert(lookup);
+
+		// create an alias to the type allowing completion
+		// of this form : `instance.BaseClass.`, which is
+		// mostly used to bypass the most derived overrides.
+		const idt = lookup.breadcrumbs.back;
+		if (!idt.length)
+			return;
+		SemanticSymbol* symbol = allocateSemanticSymbol(
+			idt, CompletionKind.aliasName, symbolFile, bc.tokens[0].index);
```

Comment:
see #136 about the last parameter. Reolution works more or less with 0 and not with `currentScope.startLocation`.
But even with 0, the resolution stops working if everything is located in a function, for example

```d
void func()
{
    interface D
    {
        void bar();
    }

    class C : D
    {
    }


    void main()
    {
        auto c = new C;
        c.D.|
    }
}     
```